BioinformaticTools-inator/bio_files_processor.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = None) -> None:
+    """
+    Converts a multi-line FASTA file to a one-line format for each sequence
+     Args:
+        input_fasta (str): The path to the input FASTA file with sequences
+        output_fasta (str, optional): The path to the output FASTA file where each sequence will be written 
+                                      on one line. If there is no output_fasta, the output file will be named as 
+                                      the input file with '_oneline' appended before the extension.
+
+    Returns:
+        None: The converted FASTA file is saved to disk
+    """
+    if output_fasta is None:
+        base, ext = os.path.splitext(input_fasta)
+        output_fasta = f"{base}_oneline{ext}"
+    with open(input_fasta, 'r') as infile, open(output_fasta, 'w') as outfile:
+        sequence = []
+        for line in infile:
+            line = line.strip()
+            if line.startswith(">"):
+                if sequence:
+                    outfile.write(''.join(sequence) + '\n')
+                    sequence = []
+                outfile.write(line + '\n')
+            else:
+                sequence.append(line)
+        if sequence:
+            outfile.write(''.join(sequence) + '\n')
+    print(f"Converted FASTA file saved to: {output_fasta}")
+
```

Comment:
можно добавить проверку на существование входного файла, чтобы скрипт не падал с ошибкой
```suggestion
if not os.path.exists(input_fasta):
    raise FileNotFoundError(f"Input file '{input_fasta}' not found.")
```